src/index.html

```diff
@@ -10,10 +10,280 @@
+      <div class="header-logo">
+        <a href="#"><img src="images/logo.png" alt="mojo"></a>
+      </div>
+      <nav class="nav">
+        <ul class="menu">
+          <li class="menu-item">
+            <a class="menu-link upper-registr is-active" href="#">apple</a>
+          </li>
+          <li class="menu-item">
+            <a class="menu-link upper-registr" href="#">samsung</a>
+          </li>
+          <li class="menu-item upper-registr">
+            <a class="menu-link upper-registr" href="#">Смартфоны и телефоны</a>
+          </li>
+          <li class="menu-item">
+            <a class="menu-link upper-registr" href="#" data-qa="nav-hover">Ноутбуки и компьютеры</a>
+          </li>
+          <li class="menu-item">
+            <a class="menu-link upper-registr" href="#">гаджеты</a>
+          </li>
+          <li class="menu-item">
+            <a class="menu-link upper-registr" href="#">планшеты</a>
+          </li>
+          <li class="menu-item">
+            <a class="menu-link upper-registr" href="#">фото</a>
+          </li>
+          <li class="menu-item">
+            <a class="menu-link upper-registr" href="#">видео</a>
+          </li>
+      </ul>
+    </nav>
+  </header>
+  <main class="main">
+    <div class="catalog-card">
+      <div class="product-card" data-qa="card">
+        <div class="product-img">
+          <img class="product-icon" src="./images/imac.jpeg" alt="product_card">
+        </div>
+      <div class="product-info">
+        <p class="product-info-price"><span class="bold-letter">Моноблок <span class="upper-registr">apple a1419</span>
+          iMac 27" Retina 5K <span class="upper-registr">(mned2ua/a)</span></span></p>
+        <p class="product-info-price-key">Код товара: 195434</p>
+      </div>
+      <div class="product-character-info">
+        <div class="stars-info">
+            <img class="star-img" src="./images/star-active.svg" alt="">
+            <img class="star-img" src="./images/star-active.svg" alt="">
+            <img class="star-img" src="./images/star-active.svg" alt="">
+            <img class="star-img" src="./images/star-active.svg" alt="">
+            <img class="star-img" src="./images/star.svg" alt="">
+        </div>
+        <div class="review">
+          <p class="text-review">Отзывов: 5</p>
+        </div>
+      </div>
+      <div class="price-info">
+        <p class="price-info-text">Цена:</p>
+        <p class="price-info-text"><span class="bold-letter text-price">69 999 грн</span></p>
+      </div>
+      <div class="product-buy">
+        <a href="#" class="button-buy-product" data-qa="card-hover">купить</a>
+      </div>
+    </div>
+    <div class="product-card">
+        <div class="product-img">
+          <img class="product-icon" src="./images/imac.jpeg" alt="product_card">
+        </div>
+      <div class="product-info">
+        <p class="product-info-price"><span class="bold-letter">Моноблок <span class="upper-registr">apple a1419</span>
```

Comment:
make heading tag here and pay attention to nesting tags (they must be placed on the new line)